pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -25,8 +26,28 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	"k8s.io/kubernetes/pkg/kubelet/network"
 	"k8s.io/kubernetes/pkg/kubelet/types"
+	"k8s.io/kubernetes/pkg/kubelet/util/format"
 )
 
+// createPodSandbox creates a pod sandbox and returns (podSandBoxID, message, error).
+func (m *kubeGenericRuntimeManager) createPodSandbox(pod *api.Pod, podIP string, attempt uint32) (string, string, error) {
```

Comment:
we can remove `podIP` as a parameter here.

It's used a bit down the line in `GenerateRunContainerOptions`, but it's always blank for that anyways in the current implementation so we can just add an empty string back in for it later in this function and hide it from the caller.